CHANGELOG.md

```diff
@@ -1,4 +1,20 @@
 # Yellowbox Changelog
+## 0.7.1
+### Added
+* pytest integration, with the `docker_client` fixture
+* `AsyncRunMixin` class, to allow starting up services asynchronously
+  * Note: Currently, only waiting for services to start is asynchronous.
+* `webserver.verbose_http_side_effect` to print all http traffic through an endpoint.
+* Webserver: all endpoint factory functions now also accept a `name` parameter, to manually set the name of an endpoint.
+* `RetrySpec.aretry` to retry a function but to wait asynchronously between attempts. 
+### Changed
+* `docker_client` renamed to `open_docker_client` (legacy alias kept for backwards compatibility)
```

Comment:
a bit confusing because you wrote about the new purest fixture with that name, maybe add that you are referring to the docker_client function?